NWHarvest.Web/Views/Growers/_GrowerListing.cshtml

```diff
@@ -3,7 +3,7 @@
 @{ 
     var availableListings = Model.Where(l => l.IsAvailable == true && l.ExpirationDate > DateTime.UtcNow);
     var pendingPickupListings = Model.Where(l => l.IsAvailable == false && l.IsPickedUp == false && l.ExpirationDate > DateTime.UtcNow);
-    var claimedOrExpiredListings = Model.Where(l => l.IsPickedUp == true || (l.IsAvailable == false && l.IsPickedUp == false && l.ExpirationDate < DateTime.UtcNow));
+    var claimedOrExpiredListings = Model.Where(l => l.IsPickedUp == true || (l.IsAvailable == true && l.IsPickedUp == false && l.ExpirationDate < DateTime.UtcNow));
```

Comment:
Can we simplify this to 
`var claimedOrExpiredListings = Model.Where(l => l.IsPickedUp || l.ExpirationDate < DateTime.UtcNow);`
?

Reply:
Yep.